code/game/gamemodes/cult/runes.dm

```diff
@@ -407,7 +407,7 @@ structure_check() searches for nearby cultist structures required for the invoca
 				to_chat(M, "<span class='cultlarge'>\"I accept this sacrifice.\"</span>")
 			else
 				to_chat(M, "<span class='cultlarge'>\"I accept this meager sacrifice.\"</span>")
-	playsound(offering, 'sound/misc/demon_consume.ogg', 100, TRUE, SOUND_RANGE_SET(10))
+	playsound(offering, 'sound/effects/cultsacrifice.ogg', 100, TRUE, SOUND_RANGE_SET(10))
```

Comment:
video? (or is this not intentional)

Reply:
This was an AA suggestion, I tested it but probably can't get a pure video of it